File: bundle/base/apps/demo/server/services/samples/javascript/remote-put-json.js

/**
 *
 *  EN: Service REST with PUT
 *  EN: Let's call an external REST service with the method
 *  EN: put and send a JSON.
 *
 *  PT: Serviço REST com PUT
 *  PT: Vamos chamar um serviço REST externo com o método
 *  PT: put e enviar um JSON.
 *
 */

var client = _remote.init()

var response = client.asJSON().put(
    'http://httpbin.org/put',
    _val.init()
        .set("name", "morpheus")
        .set("job", "leader")
)

_out.json(response)

File: bundle/base/apps/demo/server/services/samples/python/remote-put-json.py

#
#  EN: Service REST with PUT
#  EN: Let's call an external REST service with the method
#  EN: put and send a JSON.
#
#  PT: Serviço REST com PUT
#  PT: Vamos chamar um serviço REST externo com o método
#  PT: put e enviar um JSON.
#

client = _remote.init()

response = client.asJSON().put(
    "http://httpbin.org/put",
    _val.map()
        .set("name", "morpheus")
        .set("job", "leader")
)

_out.json(response)
